audius-cli

```diff
@@ -373,11 +374,12 @@ def pull(ctx, branch):
 def upgrade(ctx, branch):
     """Pull and restarts all running services"""
     ctx.forward(pull)
-    ctx.invoke(restart) # i think this should call launch (or just do docker compose up)
+    ctx.invoke(restart)
 
 
 @cli.command()
-@click.argument("cron-expression", default=f"{random.randint(0, 60)} * * * *") # random so nodes in the network stagger upgrades
+# random so nodes in the network stagger upgrades
+@click.argument("cron-expression", default=f"{random.randint(0, 59)} */6 * * *")
```

Comment:
@cheran-senthil i made this every hour on purpose so nodes don't wait 6 hours to consume the latest upgrade